### ISSUE TITLE
[Bug] clears current terminal instead of rspec's

### ISSUE THREAD

**Issue by VEShved** [noku/rails-run-spec-vscode#26]
[Here](https://github.com/jemmyw/rails-run-spec-vscode/blob/master/src/terminal.ts#L66) is the sending of clear command. But it sends it to current terminal. If I am working with two or more terminals it is very bad not to control clearing.

**Comment by jhirn** [comment]
Agreed. Might try fixing this via PR. I'd actually be happier with Scroll to bottom of terminal over clear.
